crypto/keys/pem/pem_test.go

```diff
@@ -91,10 +91,10 @@ func TestFromProto(t *testing.T) {
 		},
 	} {
 		signer, err := FromProto(test.keyProto)
-		if gotErr := err != nil; gotErr != test.wantErr {
+		if (err != nil) != test.wantErr {
 			t.Errorf("%v: FromProto(%#v) = (_, %q), want (_, nil)", test.desc, test.keyProto, err)
```

Comment:
This error message looks wrong - it assumes that we wanted `err == nil`, but that's not the case if `test.wantErr == true`.

Reply:
Reverted this change.